app/controllers/items_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ItemsController < ApplicationController
+  before_action :set_item, only: %i[show update destroy]
+
+  # GET /items
+  def index
+    @items = Item.all
+
+    render json: @items
+  end
+
+  # GET /items/1
+  def show
+    render json: @item
+  end
+
+  # POST /items
+  def create
+    @item = Item.new(item_params)
+
+    if @item.save
+      render json: @item, status: :created, location: @item
+    else
+      render json: @item.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /items/1
+  def update
+    if @item.update(item_params)
+      render json: @item
+    else
+      render json: @item.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /items/1
+  def destroy
+    @item.destroy
```

Comment:
Additionally, the reason this may be having some issues is due to the fact that reserves is a dependent model - however, yopu did specificy this in model. Let me know if you may need further assitance on this, I can definitely give you guys 15 minutes of my time and jump on a call with you guys. Email me at mayacdouglas@gmail.com or reach out on slack.